frontend/src/routes/rooms/[roomId]/RoomHeader.svelte

```diff
@@ -17,15 +35,25 @@
 </script>
 
 <header class="mt-8">
-  Room URL: <span class="whitespace-nowrap">{url}</span>
+  {#if canShare}
+    <TgButton
+      id="shareRoomButton"
+      type="primary"
+      class="m-2 p-1 bg-white text-black shadow"
+      on:click={handleShareRoom}><TgShare /></TgButton
+    >
+    Room URL: <span class="whitespace-nowrap">{url}</span>
+  {:else}
+    Room URL: <span class="whitespace-nowrap">{url}</span>
+  {/if}
```

Comment:
This is a little more straightforward
```suggestion
  {/if}
  Room URL: <span class="whitespace-nowrap">{url}</span>
```